core/controllers/suggestion.py

```diff
@@ -83,13 +89,48 @@ class SuggestionHandler(base.BaseHandler):
 
     @acl_decorators.can_suggest_changes
     def post(self):
+        change_dict = self.payload.get('change')
+        target_id = self.payload.get('target_id')
+        target_type = self.payload.get('target_type')
+        html_list = []
+        if target_type == suggestion_models.TARGET_TYPE_SKILL:
+            state_dict = change_dict['question_dict']['question_state_data']
+            state = state_domain.State.from_dict(state_dict)
+            html_list = state.get_all_html_content_strings()
+        filenames = (
+            html_cleaner.get_image_filenames_from_html_strings(html_list))
+        if len(filenames) > 0:
+            suggestion_image_entity_type = (
+                fs_services.get_entity_type_for_suggestion_target(target_type))
+        for filename in filenames:
+            image = self.request.get(filename)
+            if not image:
```

Comment:
Make sure that the EditExplorationContentSuggestion is working as expected!

Reply:
Yeah , I've it works fine